package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -1,22 +1,27 @@
 # Changes in usd-unitysdk for Unity
 
 ## [Unreleased] - 2020-10-27
-### Changed
-- Set material name in Unity to match the material name in the USD file on import.
+### Features
+- Updated USD version to v20.08
+- Set material name in Unity to match the material name in the USD file on import (#174).
+- Shader exporter: added PBR export support to standard shader (#191, #206).
+- Shader exporter: added support for in-memory and render texture export (#125).
+- Shader exporter: added support for texture wrap modes (#125).
+- Added USDZ export from recorder clip.
+- Added Scope import as XForm (#154).
 
 ### Bug Fixes
-- Fix the scene primMap that was corrupted with invalid prim making the write fail.
-- Fix an inability to refresh layers in Editor Mode
+- Fixed a crash happening when calling Scene.Read on non existing prims (#158).
+- Fixed an inability to refresh layers in Editor Mode (#155).
 - Fixed USDPayload's "Is Loaded" field in the inspector staying at false even when the payload has been loaded.
 - Fixed Transform Overrides export extension (USDU-124)
```

Comment:
I know it's not part of this PR's changes but do we want to reference internal JIRA ids in the public changelog?

Reply:
Don't think so. Github issues are more useful to the user and we should have jira issue numbers in the commit messages.